src/main/java/org/reaktivity/specification/socks/internal/SocksFunctions.java

```diff
@@ -126,6 +229,57 @@ public String getPrefixName()
         }
     }
 
+    private static void fillInBytes(
+        byte[] addressBytes,
+        int index,
+        String groupHex)
+    {
+        short res = parseShort(groupHex, 16);
+        addressBytes[index++] = (byte) ((res >> 8) & 0xff);
+        addressBytes[index] = (byte) (res & 0xff);
+    }
+
+    private static void fillInIpv6HexCompressed(
```

Comment:
Rename `fillInIpv6HexCompressed` to `encodeCompressedIpv6AddressBytes`.

Reply:
done